README.md

```diff
@@ -70,22 +70,167 @@ source for the project.  Packaging and cataloging are covered below.
+  - rxy_semantic     (presentation) float64 90.0 -0.0 -0.0 ... 0.02724 -10.4
+  - ryz_semantic     (presentation) float64 -0.0 -0.0 -0.0 ... -16.89 -0.2055
+  - rxz              (presentation) float64 0.0 0.0 0.0 ... 13.22 -2.621 -14.72
+  * time_bin         (time_bin) MultiIndex
+  - time_bin_start   (time_bin) int64 70
+  - time_bin_end     (time_bin) int64 170
+Attributes:
+    stimulus_set_identifier:  dicarlo.hvm.public
+    stimulus_set:                     id                             backgrou...
+```
+
+You can also write a Data Assembly to a netCDF file with `brainio.packaging.write_netcdf()`.  
+`write_netcdf` returns the new file's SHA-1 hash.  
+
+```pycon
+>>> from brainio.packaging import write_netcdf
+>>> hvm_assy_new = hvm_assy.sel(subregion='aIT', object_name='apple')
+>>> hvm_assy_new
+<xarray.NeuronRecordingAssembly 'dicarlo.MajajHong2015.public' (neuroid: 17, presentation: 2320, time_bin: 1)>
+array([[[-0.64983755],
+        [-1.058876  ],
+        [ 1.3953547 ],
+        ...,
+        [-0.21288353],
+        [ 1.6455535 ],
+        [ 3.1486752 ]],
+       [[-0.5977455 ],
+        [ 2.806696  ],
+        [ 0.25336486],
+        ...,
+        [ 0.6877102 ],
+        [ 0.683575  ],
+        [ 2.6989598 ]]], dtype=float32)
+Coordinates:
+  * neuroid          (neuroid) MultiIndex
+  - neuroid_id       (neuroid) object 'Chabo_L_A_8_8' ... 'Chabo_L_A_6_5'
+  - arr              (neuroid) object 'A' 'A' 'A' 'A' 'A' ... 'A' 'A' 'A' 'A'
+  - col              (neuroid) int64 8 8 6 7 5 7 7 6 6 5 6 5 5 5 6 5 5
+  - hemisphere       (neuroid) object 'L' 'L' 'L' 'L' 'L' ... 'L' 'L' 'L' 'L'
+  - animal           (neuroid) object 'Chabo' 'Chabo' ... 'Chabo' 'Chabo'
+  - y                (neuroid) float64 1.4 1.8 0.6 1.4 1.0 ... -0.2 -0.2 0.2 0.6
+  - x                (neuroid) float64 1.4 1.4 0.6 1.0 0.2 ... 0.2 0.6 0.2 0.2
+  - region           (neuroid) object 'IT' 'IT' 'IT' 'IT' ... 'IT' 'IT' 'IT'
+  - row              (neuroid) int64 8 9 6 8 7 9 7 9 7 9 8 8 3 4 4 5 6
+  * presentation     (presentation) MultiIndex
+  - stimulus_id         (presentation) object '5cde85cc63e677623c606ebf6d21a6b02...
+  - repetition       (presentation) int64 18 18 18 18 19 19 ... 17 16 16 17 17
+  - stimulus         (presentation) int64 431 310 344 530 60 ... 2551 2544 5 2
+...
+  - ryz_semantic     (presentation) float64 -0.0 -0.0 -0.0 ... 42.79 -0.2055
+  - rxz              (presentation) float64 0.0 0.0 0.0 ... -23.87 -3.039 -14.72
+  * time_bin         (time_bin) MultiIndex
+  - time_bin_start   (time_bin) int64 70
+  - time_bin_end     (time_bin) int64 170
+Attributes:
+    stimulus_set_identifier:  dicarlo.hvm.public
+    stimulus_set:                     id                             backgrou...
+>>> write_netcdf(assembly=hvm_assy_new, target_netcdf_file='hvm_aIT_apple.nc')
+'7da262e1d35de5fc26bbbe9b10c481792cef1bde'
+```
+
+### Using Packaged and Cataloged Stimulus Sets And Data Assemblies
```

Comment:
as above, I think this needs to be first. Most users won't load anything from local files